python/setup.py

```diff
@@ -61,11 +63,15 @@ def CheckCUDA():
     "-lknowhere",
     "-Wl,-rpath,$ORIGIN/../../../",
 ]
+
 SWIG_OPTS = [
     "-c++",
     "-I" + os.path.join(KNOWHERE_ROOT, "knowhere"),
 ]
 
+if is_cuda_ver:
+    SWIG_OPTS.append("-DKNOWHERE_GPU_VERSION=1")
```

Comment:
better use:
SWIG_OPTS.append("-DKNOWHERE_GPU_VERSION=ON")

Reply:
number is better, ON used in cmake option.